src/main/kotlin/no/nav/familie/ef/sak/infotrygd/PeriodeService.kt

```diff
@@ -24,15 +24,15 @@ class PeriodeService(
         private val replikaClient: InfotrygdReplikaClient
 ) {
 
-    fun hentPerioderFraEfOgInfotrygd(personIdent: String): List<InternPeriode> {
+    fun hentPerioderForOvergangsstønadFraEfOgInfotrygd(personIdent: String): List<InternPeriode> {
```

Comment:
Hvis vi vil unngå å ha "Overgangsstønad" i alle metodenavnene så kan vi jo sende inn stønadstype til funksjonene slik at de faktisk plukker ut litt mer generisk...?

Reply:
Og at controlleren har overgangsstønad i input til servicen?

Reply:
Ja, enig i det. Vurderte det først men innså men når oppgaven min ble endred så tenkte jeg att jeg likevel bare gjør det litt tydligere nå 👼 
Eller tydligere.. Men for å unngå total missforståelse senere :o